tools/updater/updater.go

```diff
@@ -112,28 +113,25 @@ func updateJazzerNpm(path string, version string) {
 	cmd.Dir = path
 	cmd.Stderr = os.Stderr
 	cmd.Stdout = os.Stdout
-	err := cmd.Run()
-	if err != nil {
-		log.Error(err)
-		os.Exit(1)
-	}
+	handleErr(cmd.Run())
```

Comment:
Personal taste, but I'd prefer:

```suggestion
    err := cmd.Run()
    handleErr(err)
```